alpha/apps/kaltura/modules/extwidget/actions/playManifestAction.class.php

```diff
@@ -611,6 +626,8 @@ protected function initFlavorAssetArray()
 
 		if ($flavorByTags)
 		{
+			if(self::shouldAddAltAudioFlavors(($this->deliveryAttributes->getFormat())))
```

Comment:
Why not add these tags one in the execute function?
If tags where sent by the user than get them and call the function otherwise add this support in to the "getDefaultTagsByFormat" function